cores/arduino/HardwareTimer.cpp

```diff
@@ -927,6 +929,14 @@ void HardwareTimer::detachInterrupt(uint32_t channel)
   // Disable interrupt corresponding to channel and clear callback
   __HAL_TIM_DISABLE_IT(&(_timerObj.handle), interrupt);
   callbacks[channel] = NULL;
+  args[channel] = NULL;
+}
+
+void* HardwareTimer::getArg(uint32_t channel){
```

Comment:
Can you add a comment about this new function with brief, parameter and return value (like other functions)